gateway/src/apicast/policy/soap/policy.lua

```diff
@@ -0,0 +1,68 @@
+--- SOAP Policy
+-- This policy adds support for a very small subset of SOAP.
+-- This policy basically expects a SOAPAction URI in the SOAPAction header or
+-- the content-type header.
+-- The SOAPAction header is used in v1.1:
```

Comment:
"....v1.1 of the SOAP Standard"  
(just to avoid any confusion in readers about versions of apicast or something?)

Reply:
:+1: